<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="HAIDAR" failures="1" tests="3" name="CucumberRunner" time="9.303" errors="0" timestamp="2025-03-29T23:01:38 IST">
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="4.997"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="4.997"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="3.407">
    <failure type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError
at org.junit.Assert.fail(Assert.java:87)
at org.junit.Assert.assertTrue(Assert.java:42)
at org.junit.Assert.assertFalse(Assert.java:65)
at org.junit.Assert.assertFalse(Assert.java:75)
at org.haidar.cucumberbdd.StepDefinitions.i_click_the_login_button(StepDefinitions.java:89)
at âœ½.I click the login button(file:///D:/EclipseSoftware/cucumberbddwithTestNG/src/test/resources/featuresToBeRunInParallel/login.feature:13)
]]>
    </failure>
  </testcase> <!-- runScenario -->
</testsuite> <!-- CucumberRunner -->
